test/inductor/test_torchinductor.py

```diff
@@ -611,6 +611,15 @@ def fn(x, y):
 
         self.common(fn, (x, y))
 
+    def test_add_complex(self):
```

Comment:
Add tests for:
1) complex + non-complex
2) complex + scalar
3) `alpha=...`

Reply:
Also if you have a test that demonstrates the aliasing issue with the prior version you should add that too.

Reply:
The logic in decomposition only supports complex+complex without alpha. The use of alpha introduces complex multiplication, which maybe we can handle separately?

Reply:
Added test for alpha